build.assets/genproto.sh

```diff
@@ -10,9 +10,8 @@ main() {
   # Generated protos are written to
   # <teleport-root>/github.com/gravitational/teleport/..., so we copy them to
   # the correct relative path.
-  trap 'rm -fr github.com' EXIT   # don't leave github.com/ behind
-  # cleanup gen/proto folders
-  rm -fr api/gen/proto gen/proto lib/teleterm/api/protogen lib/prehog/gen lib/prehog/gen-js
+  trap 'rm -fr github.com' EXIT  # don't leave github.com/ behind
+  rm -fr api/gen/proto gen/proto # cleanup gen/proto folders
```

Comment:
No `lib/prehog/gen{,-js}` in the `rm` looks wrong?

Reply:
> No `lib/prehog/gen{,-js}` in the `rm` looks wrong?

Everything is generated to `gen/proto` now so we no longer need to clean up `gen` directories in selected libraries.

Reply:
Right!